packages/cli/src/lingui-extract.ts

```diff
@@ -23,12 +23,6 @@ export default async function command(
   config: LinguiConfigNormalized,
   options: Partial<CliExtractOptions>
 ): Promise<boolean> {
-  // `react-app` babel plugin used by CRA requires either BABEL_ENV or NODE_ENV to be
-  // set. We're setting it here, because lingui macros are going to use them as well.
-  if (!process.env.BABEL_ENV && !process.env.NODE_ENV) {
-    process.env.BABEL_ENV = "development"
-  }
-
```

Comment:
this is not needed

Reply:
Shouldn't this be removed also from the `lingui-extract-template.ts`?

Reply:
done